maintainer/docker/opensuse/Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM opensuse:42.1
+FROM opensuse:42.3
```

Comment:
I did that on purpose so the version is fixed, just like with ubuntu (16.04), debian (8) and centos (7).